cumulus/test/service/src/cli.rs

```diff
@@ -45,9 +45,6 @@ pub struct TestCollatorCli {
 	#[arg(raw = true)]
 	pub relaychain_args: Vec<String>,
 
-	#[arg(long)]
-	pub use_null_consensus: bool,
```

Comment:
But I see this still in use here? https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/zombienet/zombienet-sdk/tests/zombie_ci/pov_recovery.rs#L194-L194

Reply:
Will have a look at the zombienet test later, never know when these tests are expected to fail or not. Have seen them failing in other PRs